integration/kubernetes/runtimeclass_workloads/pod-footloose.yaml

```diff
@@ -0,0 +1,58 @@
+#
+# Copyright (c) 2019 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+apiVersion: v1
+kind: Pod
+metadata:
+  name: footclr
+spec:
+  runtimeClassName: kata
+  volumes:
+  - name: runv
+    emptyDir:
+      medium: "Memory"
+  - name: runlockv
+    emptyDir:
+      medium: "Memory"
+  - name: tmpv
+    emptyDir:
+      medium: "Memory"
+  - name: fakecgroup
+    hostPath:
+      path: /sys/fs/cgroup
+  - name: ssh-dir
+    emptyDir:
+      medium: "Memory"
+  - name: ssh-config-map
+    configMap:
+      name: ssh-config-map
+      defaultMode: 384
+  containers:
+  - name: vmcontainer
+    image: quay.io/footloose/clearlinux:latest
```

Comment:
@GabyCT - Oops - my bad! Seeing https://github.com/kata-containers/packaging/issues/514 has made me realise - we can't use CL here as that makes this test architecture-specific!

Reply:
I think we can use `quay.io/footloose/ubuntu18.04` as it is an LTS version and should be there for a lot of time